### DISCUSSION TITLE
Orthomosaic and DEM Issues

### DISCUSSION THREAD

**Discussion by coolbeansnoah** [cnr-isti-vclab/TagLab#129] [General]
I am trying to create a project with the DEM, but I keep getting this problem. 
![image](https://github.com/cnr-isti-vclab/TagLab/assets/73969452/3356fc4f-761e-494c-bc46-b3fde86d5c93)
I'm exporting my orthomosaics and DEM from Agisoft Metashape, I would love to know if there are some settings I can change while exporting the maps to not have this problem. 
![image](https://github.com/cnr-isti-vclab/TagLab/assets/73969452/fbd47199-fcea-4a70-8188-05e77da2ed85)
So far I have tried to adjust the max pixel dimensions and increase the resolution of the DEM but there still seems to be a few pixels off.

**Comment by Jordan-Pierce** [comment]
Hi @coolbeansnoah can you confirm that when you're exporting both the DEM and Orthomosaic from Metashape, the final values representing the dimensions are the same? Although they are the same in Metashape, when exporting, there's a chance they're a few pixels off (even when made from the same surface) if you don't set the boundaries correctly within the export panel (it should be a radio box that adjusts the dimensions).

**Comment by coolbeansnoah** [comment]
When export in metashape I create a shape and set as a boundary and then clip to boundary when exporting. This is just a solution I have come up with, not to set on keeping this method. Do you have any suggestions or different methods?

**Comment by Jordan-Pierce** [comment]
@coolbeansnoah regardless of the shape, your Orthomosaic and DEM are off by a single pixel. When exporting, try toggling the 
setup boundaries radio box and see how that adjusts the dimensions of the exported raster. You need to keep track of the DEM and Orthomosaic so they are exported with the same dimensions

![6yfXRrn4-1Rr58JZiX4uM03hAL3jQgv6Aw](https://github.com/cnr-isti-vclab/TagLab/assets/115024024/834c62c2-7764-4fc5-9433-4f195c111e41)
